### ISSUE TITLE
Application does not gracefully handle glob patterns in bash

### ISSUE THREAD

**Issue by benjrichardson** [cheddarbread/sheetloaf#3]
`node . test/samples/styles/*.scss css --style compressed --no-source-map --use autoprefixer`

What the application gets from process.argv

```
[ '/usr/bin/node',
  '/home/ben/sheetloaf',
  'test/samples/styles/multi.scss',
  'test/samples/styles/thing.scss',
  'css',
  '--style',
  'compressed',
  '--no-source-map',
  '--use',
  'autoprefixer' ]
```

Bash interprets the glob pattern ahead of time and gives us two arguments. The application then takes 'test/samples/styles/multi.scss', and believes 'test/samples/styles/thing.scss' to be the destination.

**Comment by benjrichardson** [comment]
A workaround for this in the meantime is to quote glob strings in the npm script, like so:

`node . test/samples/styles/*.scss css --style compressed --no-source-map --use autoprefixer`

That way your shell won't expand those patterns.

**Comment by benjrichardson** [comment]
This fixed as of 0.4.
